packages/contracts/contracts/votings/majority-voting/WhitelistVoting.sol

```diff
@@ -0,0 +1,175 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity 0.8.10;
+
+import "./MajorityVoting.sol";
+
+contract WhitelistVoting is MajorityVoting {
```

Comment:
natspec.. resp file header.. commented this also on the other files.. I would just add Giorgi, me, and you to the authors list.

Reply:
Added